pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `minimumCanEqual`           | Validation    | 01                 | 03      | Replaced by `exclusiveMinimum`                                                               |
+| `minContains`               | Validation    | 2019-09            | No      | No                                                                                           |
+| `minProperties`             | Validation    | 04                 | No      | No                                                                                           |
+| `multipleOf`                | Validation    | 04                 | No      | Replaced `divisibleBy`                                                                       |
+| `not`                       | Core          | 04                 | No      | No                                                                                           |
+| `oneOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `optional`                  | Core          | 02                 | No      | Replaced by `required`                                                                       |
+| `pattern`                   | Core          | 01                 | No      | No                                                                                           |
+| `patternProperties`         | Core          | 03                 | No      | No                                                                                           |
+| `prefixItems`               | Core          | 2020-12            | No      | Replaced `items`                                                                             |
+| `propertyNames`             | Core          | 06                 | No      | No                                                                                           |
+| `readOnly`                  | Validation    | 01                 | No      | No                                                                                           |
+| `required`                  | Validation    | 03                 | No      | No                                                                                           |
+| `requires`                  | Core          | 01                 | 03      | Replaced by `dependencies`                                                                   |
+| `title`                     | Validation    | 01                 | No      | No                                                                                           |
+| `then`                      | Core          | 07                 | No      | No                                                                                           |
+| `type` (legacy)             | Core          | 01                 | 04      | Replaced by `type`                                                                           |
```

Comment:
This is also weird. I think it's confusing to say `type` was replaced by `type`. I think the main change you want to document here is that previously, `type` could be set to a schema

Reply:
The `any` type was previously valid too, and now it's not. Maybe worth adding a note

Reply:
@jviotti How best would you say to represent `type`?

Reply:
To be honest, I'm not sure. Maybe this is OK, and we can just clarify that `type` no longer takes `any` and subschemas in its array form on the notes?

Reply:
I think this depends on how much we can put in the "Changed" column. If a brief description of the change can fit in that column, we don't need two entries. The reason I suggested two entries was because I didn't think that was an option. So, I recommend either a description of the changes in "Changes" _or_ two entries, but not both.